backend/handlers/handlers.go

```diff
@@ -191,6 +191,7 @@ func NewHandlers(
 	getAPIRouter(apiRouter)("/certs/check", handlers.postCertsCheckHandler).Methods("POST")
 	getAPIRouter(apiRouter)("/bitboxbases/connectbase", handlers.postConnectBaseHandler).Methods("POST")
 	getAPIRouter(apiRouter)("/bitboxbases/disconnectbase", handlers.postDisconnectBaseHandler).Methods("POST")
+	getAPIRouter(apiRouter)("/bitboxbases/connectElectrum", handlers.postConnectElectrumHandler).Methods("POST")
```

Comment:
consistency: `connect-electrum`.

Also, why move the endpoint into `bitboxbase/handlers`? It acts on a single base after all, right?